compose/install.md

```diff
@@ -132,7 +132,7 @@ also included below.
 1.  Run this command to download the current stable release of Docker Compose:
 
     ```console
-    $ sudo curl -L "https://github.com/docker/compose/releases/download/{{site.compose_version}}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+    $ sudo curl -L "https://github.com/docker/compose/releases/download/v2.3.3/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
@NaincyKumariKnoldus This is the installation instruction for Compose version 1. Please see the note below for instructions on how to install Compose v2 for Linux.

Reply:
We are in the process of reorganising the Compose Docs. This should resolve any confusion users have on installing Compose version 2. Closing the PR here for that reason. Feel free to continue the conversation after I close